Bottom: 3639f29f7e383980f020787070bc490ff47cd5af
Top:    c982e1e5c2f37fca403544d2f79ceb515ecbcf4c
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-05 11:44:40 -0500

gcc-4.6 detects bad assembly in Linux 3.3, use Linux 3.3.8

---

diff --git a/.farmrc b/.farmrc
index 88a03a8..1e543cf 100644
--- a/.farmrc
+++ b/.farmrc
@@ -4,10 +4,10 @@ linuxgit=$HOME/src/linux
 tags="v2.6.14 v2.6.15 v2.6.16 v2.6.17 v2.6.18 v2.6.19 v2.6.20 v2.6.21
 v2.6.22 v2.6.23 v2.6.24 v2.6.25 v2.6.26 v2.6.27 v2.6.28 v2.6.29 v2.6.30
 v2.6.31 v2.6.32 v2.6.33 v2.6.34 v2.6.35 v2.6.36 v2.6.37 v2.6.38 v2.6.39
-v3.0 v3.1 v3.2 v3.3 v3.4 v3.5 v3.6 v3.7 v3.8 v3.9 v3.10 v3.11 v3.12
+v3.0 v3.1 v3.2 v3.3.8 v3.4 v3.5 v3.6 v3.7 v3.8 v3.9 v3.10 v3.11 v3.12
 linux-next/master"
 make="make -j11"
-kcc=gcc-4.4
+kcc=gcc
 ucc=gcc
 ucflags=-DPATH_MAX=4096
 export EXTRA_CFLAGS=-Werror-implicit-function-declaration
